tests/functional/api/groups/test_members.py

```diff
@@ -195,62 +159,16 @@ def test_it_removes_authed_user_from_group(
     ):
 
         group_member, token = group_member_with_token
-        headers = {"Authorization": "Bearer {}".format(token.value)}
+        headers = {"Authorization": f"Bearer {token.value}"}
 
-        app.delete("/api/groups/{}/members/me".format(group.pubid), headers=headers)
+        app.delete(f"/api/groups/{group.pubid}/members/me", headers=headers)
 
         # We currently have no elegant way to check this via the API, but in a
         # future version we should be able to make a GET request here for the
         # group information and check it 404s
         assert group_member not in group.members
 
 
-@pytest.fixture
-def user(db_session, factories):
-    user = factories.User(authority="example.com")
-    db_session.commit()
-    return user
-
-
-@pytest.fixture
-def third_party_user(db_session, factories):
-    user = factories.User(authority="thirdparty.com")
-    db_session.commit()
-    return user
-
-
-@pytest.fixture
-def auth_client(db_session, factories):
-    auth_client = factories.ConfidentialAuthClient(
-        authority="thirdparty.com", grant_type=GrantType.client_credentials
-    )
```

Comment:
Note here that `auth_client` is paired with the `third_party_user` not the `user` as in the previous test.